packages/core/src/spec/lib/jdk/JdkHelperSpec.ts

```diff
@@ -63,4 +64,25 @@ describe('JdkHelper', () => {
       expect(env['JAVA_HOME']).toBe('C:\\Users\\user\\jdk8\\');
     });
   });
+
+  describe('validatePath', () => {
+    it('Checks that given a valid path, validatePath will pass', async () => {
```

Comment:
Can you add tests for the path validation each platform, similar to what the `getEnv()` test does?

Reply:
I can, but if `getJavaHome()` it proven to be working those tests will mean nothing.. `validatePath` uses `getJavaHome` and after we got to this folder it all looks the same

Reply:
Sounds good, adding tests to `getJavaHome()` should be enough.